packages/chart/src/components/SparkLine.jsx

```diff
@@ -72,7 +72,7 @@ export default function SparkLine({ width: parentWidth, height: parentHeight })
   
   // Just do this once up front otherwise we end up 
   // calling clean several times on same set of data (TT)
-  const cleanedData = cleanData(data);
+  const cleanedData = cleanData(data,"Date");
```

Comment:
@tturnerswdev33 - this looks like the key your trying to exclude and should probably be dynamic, for example if its coming from the xAxis as "Date" use something like `config.xAxis.dataKey` instead of "Date".

Reply:
New commit fixes this